### PR TITLE
onboarding ui - authentication page added

### PR DESCRIPTION
**Pull request by ShakirHajabba**
OnBoarding-UI-SignIn

### CODE REVIEW

File: src/components/Authentication/authBody/authBody.component.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import SignIn from './authForm/signIn.component';
+
+function AuthBody() : JSX.Element {
+
+  const auth : AuthStateType = useSelector((state : StoreType) => state.auth);
+
+  return (
+    <div className={styles.authBody}>
+      <div className={styles.authBody__googleSignIn}>
+          <img className={styles.googleIcon} src="https://developers-dot-devsite-v2-prod.appspot.com/identity/sign-in/g-normal.png" alt="" />
```
Comment (prateekoli9) [inline src/components/Authentication/authBody/authBody.component.tsx:15]: Save this as an asset and use that here

File: src/components/Authentication/authBody/authForm/forgotPassword.component.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (prateekoli9) [inline src/components/Authentication/authBody/authForm/forgotPassword.component.tsx:1]: Move to authBody -> ForgotPassword folder

File: src/components/Authentication/authBody/authForm/joinNow.component.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { useDispatch } from 'react-redux';
```
Comment (prateekoli9) [inline src/components/Authentication/authBody/authForm/joinNow.component.tsx:1]: Move to authBody -> JoinNow folder

File: src/components/Authentication/authBody/authForm/loginWithOtp.component.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (prateekoli9) [inline src/components/Authentication/authBody/authForm/loginWithOtp.component.tsx:1]: Move to Authentication -> OtpLogin folder

File: src/components/Authentication/authBody/authForm/signIn.component.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import text from 'helpers/text.helper';
```
Comment (prateekoli9) [inline src/components/Authentication/authBody/authForm/signIn.component.tsx:1]: move to authBody -> SignIn folder

File: src/components/Authentication/authHeader/authHeader.component.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './authHeader.module.scss';
+import text from 'helpers/text.helper';
+
+function AuthHeader() : JSX.Element {
+  return (
+    <div className={styles.authHead}>
+      <div className={styles.authHead__jayporeLogo}>
+        <img src="https://www.jaypore.com/images/jaypore_logo_retina.png" alt="" />
```
Comment (prateekoli9) [inline src/components/Authentication/authHeader/authHeader.component.tsx:9]: Download image as an asset

File: src/components/header/header.component.tsx [hunk 7]
```diff
@@ -1,14 +1,23 @@
 import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 // import {ROUTES} from '../../helpers/routes.helper'
```
Comment (prateekoli9) [inline src/components/header/header.component.tsx:3]: Remove commented code

File: src/store/reducers/authentication.ts [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    isOtpLogin : false,
+    isJoinNow : false,
+};
+
+export default (state = initialState, action): AuthStateType => {
+  
+  
+  switch (action.type) {
+    case TOGGLE_AUTH:
+      return { ...state, isAuthOpen: action.payload, isForgotPwd : false, isJoinNow : false, isOtpLogin: false, isSignIn : true};
```
Comment (prateekoli9) [inline src/store/reducers/authentication.ts:18]: change to ...initialState

File: src/types/store.types.ts [hunk 9]
```diff
@@ -1,5 +1,15 @@
 import { UserState } from './user.types';
 
-export type Store = {
+export type StoreType = {
```
Comment (prateekoli9) [inline src/types/store.types.ts:3]: Rename it to Store

File: src/types/store.types.ts [hunk 10]
```diff
@@ -1,5 +1,15 @@
...
 
-export type Store = {
+export type StoreType = {
     users: UserState;
-};
\ No newline at end of file
+    auth: AuthStateType;
+};
+
+export type AuthStateType = {
```
Comment (prateekoli9) [inline src/types/store.types.ts:8]: Move to auth.types.ts

File: src/components/Authentication/authBody/authBody.component.tsx (hunk 1 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authBody/authBody.component.tsx:15]: Asset saved and used.

File: src/components/Authentication/authBody/authForm/forgotPassword.component.tsx (hunk 2 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authBody/authForm/forgotPassword.component.tsx:1]: forgotPassword.component.tsx moved to Authentication -> ForgotPassword folder

File: src/components/Authentication/authBody/authForm/joinNow.component.tsx (hunk 3 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authBody/authForm/joinNow.component.tsx:1]: joinNow.component.tsx moved to authBody -> JoinNow folder

File: src/components/Authentication/authBody/authForm/loginWithOtp.component.tsx (hunk 4 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authBody/authForm/loginWithOtp.component.tsx:1]: loginwithOtp.tsx moved to Authentication -> OtpLogin folder

File: src/components/Authentication/authBody/authForm/signIn.component.tsx (hunk 5 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authBody/authForm/signIn.component.tsx:1]: signIn.component.tsx moved to authBody -> SignIn folder

File: src/components/Authentication/authHeader/authHeader.component.tsx (hunk 6 above)
Comment (ShakirHajabba) [reply src/components/Authentication/authHeader/authHeader.component.tsx:9]: Asset saved and used.

File: src/components/header/header.component.tsx (hunk 7 above)
Comment (ShakirHajabba) [reply src/components/header/header.component.tsx:3]: commented code removed.

File: src/store/reducers/authentication.ts (hunk 8 above)
Comment (ShakirHajabba) [reply src/store/reducers/authentication.ts:18]: changes made.

File: src/types/store.types.ts (hunk 9 above)
Comment (ShakirHajabba) [reply src/types/store.types.ts:3]: StoreType renamed to Store

File: src/types/store.types.ts (hunk 10 above)
Comment (ShakirHajabba) [reply src/types/store.types.ts:8]: moved to auth.types.ts

**Pull request merged**